action.js

```diff
@@ -19,23 +83,141 @@ window.onload = function meunWindow() {
     window.onscroll = function () {};
   }
 
-  open.addEventListener('click', () => {
-    div.style.display = 'block';
+  function addModals() {
+    let modalitem = "";
+    projectsArray.forEach((project, i) => {
+      let tags = "";
+      project.tags.forEach((tag) => {
+        tags += `<span>${tag}</span>`;
+      });
+      modalitem += `
+      <div class="project-item" id="project${i + 1}">
+        <div class="item-header">
+          <div class="title">${project.title}</div>
+          <button data-close-button class="close-button">&times;</button>
+        </div>
+        <div class="item-body">
+          <div class="item-details">
+            <span>${project.company}</span>
+            <span class="item-role">${project.role}</span>
+            <span>${project.year}</span>
+          </div>
+          <div class="item-img img${i + 1}">
+            <p>image point</p>
+          </div>
+          <div class="desktp-cards">
+            <div class="desk-card">
+              <p>${project.modaldesc}</p>
+            </div>
+            <div class="desk-soc">
+              <div class="tags">
+                ${tags}
+              </div>
+              <div class="modal-links">
+                <a href="${
+                  project.sourceLink
+                }"><span>See live <img src="./images/live.png"/></span></a>
+                <a href="${
+                  project.demoLink
+                }"><span>See Source<img src="./images/giticon.png"/></span>
+                </a>
+              </div>
```

Comment:
- [ ] Great on the styles but for this project you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)design as much as possible. 
- Please place some gaps between your button's text and icon and make sure it matches the design on the template.

## Your design

![image](https://user-images.githubusercontent.com/83803414/162282541-19b5a402-92f0-453d-8efe-055dfdd3b0d1.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/162282581-76ecd8c9-d206-4221-a070-e7cdd2101564.png)